app/views/complaints/show.html.erb

```diff
@@ -24,7 +24,7 @@
             </p>
           </div>
         <% end %>
-        <div class="display-none" id="tta-activity-form">
+        <div <%= flash[:tta_errors].blank? && 'class="display-none"'.html_safe %> id="tta-activity-form">
```

Comment:
[suggestion]
I recommend using `sanitize` rather than `.html_safe`, unless if there's some reason that it won't work here

[question]
I'm not sure I'm following this part. Seems to me like this would print out something like `<div true id="tta-activity-form">` ?  Why do you even need `.html_safe`?  Maybe I'm just unfamiliar with the syntax?

Reply:
For whatever reason, I take the apidock line "It is recommended that you use sanitize instead of this method." very seriously, but I think `.html_safe` works fine here.
https://apidock.com/rails/String/html_safe

I'd prefer something like @Sgtpluck 's suggestion if you don't mind the `class=""` possibility.  I also like `<div class=<%= "display-none" if flash[:tta-errors].blank? %>`.